website/www/site/layouts/partials/section-menu/en/contribute.html

```diff
@@ -13,6 +13,7 @@
 <li><span class="section-nav-list-main-title">Contribute</span></li>
 <li><a href="/contribute/">Get started contributing</a></li>
 <li><a href="/contribute/get-help/">Get Help</a></li>
+<li><a href="/contribute/practises/">Beam community's practises</a></li>
```

Comment:
This should be updated to reflect the new name

Reply:
okay

Reply:
done